docs/pr-guide.md

```diff
@@ -66,6 +66,7 @@ Validation may fail for several reasons:
 
 ### Additional information:
   * You can list the available pipelines by adding a comment like `/azp list` or get the available commands by adding a comment like `azp help`.
+  * In the rare case the license/cla check fails to register a response, it can be rerun by issuing a GET request to `https://cla.dotnetfoundation.org/check/{org}/{repo}?pullRequest={pr_number}`. A successful response may be a redirect to `https://github.com`.
```

Comment:
> `/{org}/{repo}?`

In the context of runtime, perhaps it's better to use hardcoded values: `/dotnet/runtime?`

Reply:
```suggestion
  * In the rare case the license/cla check fails to register a response, it can be rerun by issuing a GET request to `https://cla.dotnetfoundation.org/check/dotnet/runtime?pullRequest={pr_number}`. A successful response may be a redirect to `https://github.com`.
```